pkg/status/conditions_test.go

```diff
@@ -70,7 +75,7 @@ func TestInitConditions(t *testing.T) {
 				Reason:             "Deprecated",
 			})
 
-			status.InitConditions(kyma, tcase.args.syncEnabled, tcase.args.watcherEnabled)
+			status.InitConditions(kyma, tcase.args.watcherEnabled)
```

Comment:
if you change to this way, it lacks one test case, which is kyma don't have sync label.

Reply:
Will add a testcase with missing labels